fabtests/common/shared.c

```diff
@@ -3059,6 +3064,29 @@ void ft_parse_hmem_opts(int op, char *optarg, struct ft_opts *opts)
 	}
 }
 
+void parse_opts_range(char* optarg)
+{
+       size_t start, inc, end;
+       int i;
+       int ret = sscanf(optarg, "r:%ld,%ld,%ld", &start, &inc, &end);
```

Comment:
Please don't mix functionality with variable declarations, and add a space after the declarations.
```
int i, ret;

ret = sscanf(...);
if (ret != 3) ...
```